### ISSUE TITLE
Create group message function

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/MyopicVicar#1357]
Currently done in google groups - possibly needs bringing in-house

Examples might be 'group of those working on Refinery issues', 'group of transcribers in a county', etc.

Original Strategy requirement: Ability to set up groups with specific email or role addresses allowing broadcast/group reply to emails.  

Needs revisiting in Version 2 - e.g. should this be emails, or in-system messages?

**Comment by edickens** [comment]
Yes, but needs a lot of thought and programming as people must be able to opt in and out.  But it should use the details in the UserID.  V1.6 Transcriber enhancement, or even V2.

**Comment by SteveBiggs** [comment]
Why should people be able to opt in and out? Isn't this to be used to send important emails and newsletters to volunteers? Their membership should be automatically determined by their role in FR.

**Comment by edickens** [comment]
The management messages will be by the message system, as at present.  The idea of groups, as I understand it, is so that there can be separate specialist groups, for example to decipher Latin entries.

**Comment by PatReynolds** [comment]
This is for just for groups such as the FreeREG development group, group of ipad owners who agree to test functionality, and the like.  Emails to groups such as "all transcribers", "everyone subscribed to the newsletters", etc. are different stories.

**Comment by SteveBiggs** [comment]
Pat, can I ask that you put this level of detail in the description of stories when you first raise them? Thanks.

**Comment by PatReynolds** [comment]
Sorry -was doing this all in a rush during the meeting, so will go back and amend now.

**Comment by DeniseColbert** [comment]
I've put #1460 as a blocker, but can remove that if it's not a suitable solution

**Comment by stsccfr** [comment]
Closing as needing more detail in exactly what is needed and why.
